### PR TITLE
Add org.kde.kronometer

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://invent.kde.org/utilities/kronometer/-/issues/1
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/utilities/kronometer/-/merge_requests/10

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.kde.kronometer

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by barthalion** [comment]
LGTM, but needs a review from KDE team.

**Comment by travier** [comment]
Needs one minor appdata change (comment in upstream PR).

File: org.kde.kronometer.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/kronometer/2.2.3/src/kronometer-2.2.3.tar.xz",
+                    "sha256": "75259b80dc2a074d2a1d5d11a909a599a998b570ad0b5bf995580d95c4411a16",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 17750,
+                        "stable-only": true,
+                        "url-template": "https://download.kde.org/stable/release-service/$version/src/kronometer-$version.tar.xz"
```
Comment (tsdgeos) [inline org.kde.kronometer.json:28]: this url doesn't match the one above

File: org.kde.kronometer.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kronometer",
+    "rename-icon": "kronometer",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
```
Comment (tsdgeos) [inline org.kde.kronometer.json:12]: Seems to want to access some dbus to disable the screenlocker, can you investigate?

Could not inhibit screen locker: QDBusError("org.freedesktop.DBus.Error.ServiceUnknown", "org.freedesktop.DBus.Error.ServiceUnknown")

**Comment by nedrichards** [comment]
bot, build org.kde.kronometer

**Comment by travier** [comment]
```
C95: -> org.freedesktop.ScreenSaver call org.freedesktop.ScreenSaver.Inhibit at /ScreenSaver
*HIDDEN* (ping)
C96: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C97: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C98: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
B-1: <- org.freedesktop.DBus return from C95
*REWRITTEN*
C99: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C100: -> org.freedesktop.ScreenSaver call org.freedesktop.ScreenSaver.Inhibit at /ScreenSaver
*HIDDEN* (ping)
B-1: <- org.freedesktop.DBus return from C100
*REWRITTEN*
Could not inhibit screen locker: QDBusError("org.freedesktop.DBus.Error.ServiceUnknown", "org.freedesktop.DBus.Error.ServiceUnknown")
Could not inhibit screen locker: QDBusError("org.freedesktop.DBus.Error.ServiceUnknown", "org.freedesktop.DBus.Error.ServiceUnknown")
C101: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C102: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C103: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C104: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
C105: -> all signal com.canonical.dbusmenu.ItemsPropertiesUpdated at /MenuBar/1
```

Probably need to talk to `org.freedesktop.ScreenSaver.Inhibit`

**Comment by travier** [comment]
LGTM. Thanks!

**Pull request closed without merging**